README.md

```diff
@@ -9,17 +9,24 @@
 
 ## Getting Started
 
-This repository includes files with Ruby that can be used to decode a Morse code message:
+This repository includes files with Ruby that can be used to is getting information from the library about the University:
 
 - Please run the program in a IRB enviorement
-
-## Author
+- From your terminal you have to run the following command: `ruby main.rb`
+- After running this command you will get following kind of information:
+![screenshot](./Screenshot_2.png)
+  ## Author
```

Comment:
- [ ] [OPTIONAL]You did great :+1: adding how to run your project but as said by the previous reviewer you should also add how to get your project locally 
- Here is a hint of how you may do that :point_down: 
![image](https://user-images.githubusercontent.com/75126481/143507890-73be31b5-b901-4f52-9140-55215651a28d.png)

Reply:
Done ✔